docs/resources/view.md

```diff
@@ -56,15 +60,17 @@ resource "snowflake_view" "test" {
 SQL
 }
 ```
+-> **Note** Instead of using fully_qualified_name, you can reference objects managed outside Terraform by constructing a correct ID, consult [identifiers guide](https://registry.terraform.io/providers/Snowflake-Labs/snowflake/latest/docs/guides/identifiers#new-computed-fully-qualified-name-field-in-resources).
+<!-- TODO(SNOW-1634854): include an example showing both methods-->
 
 <!-- schema generated by tfplugindocs -->
 ## Schema
 
 ### Required
 
-- `database` (String) The database in which to create the view. Don't use the | character.
-- `name` (String) Specifies the identifier for the view; must be unique for the schema in which the view is created. Don't use the | character.
-- `schema` (String) The schema in which to create the view. Don't use the | character.
+- `database` (String) The database in which to create the view. Due to technical limitations, don't use the following characters: `|`
```

Comment:
is this still true about the `|` character?

Reply:
Changed as we discussed.